src/mutations.ts

```diff
@@ -59,3 +60,34 @@ export const PASSWORD_RESET = gql`
     }
   }
 `;
+
+export const ExternalAuthenticationURL = /* GraphQL */ `
+  mutation externalAuthenticationUrl($pluginId: String!, $input: JSONString!) {
+    externalAuthenticationUrl(pluginId: $pluginId, input: $input) {
+      authenticationData
+      errors {
+        code
+        field
+        message
+      }
+    }
+  }
+`;
+
+export const ExternalObtainAccessTokens = /* GraphQL */ `
+  mutation AuthObtainAccessToken($pluginId: String!, $input: JSONString!) {
+    externalObtainAccessTokens(pluginId: $pluginId, input: $input) {
+      token
+      refreshToken
+      user {
+        id
+        email
+      }
+      errors {
+        field
+        code
+        message
+      }
+    }
+  }
+`;
```

Comment:
These two queries are strings but all other queries and fragments are `DocumentNode`. Why? It's inconsistent.